packages/manager/modules/dbaas-logs/src/logs/translations/Messages_fr_FR.json

```
@@ -71,21 +71,23 @@
   "logs_order_selection_title": "Choisissez l’option Logs Data Platform qui correspond à vos besoins",
   "logs_order_selection_subtitle": "Gérez, sécurisez et ne payez que ce que vous utilisez",
   "logs_order_selection_details_standard_title": "Standard",
-  "logs_order_selection_details_standard_description": "Inclus gratuitement et de façon illimitée&nbsp;:",
+  "logs_order_selection_details_standard_description": "Inclus de façon illimitée&nbsp;:",
   "logs_order_selection_details_standard_description_1": "tableau de bord et flux Graylog",
   "logs_order_selection_details_standard_description_2": "contrôle d’accès à base de rôles",
   "logs_order_selection_details_standard_description_3": "alias Elasticsearch",
   "logs_order_selection_details_standard_description_4": "appels API Graylog et Elasticsearch",
   "logs_order_selection_details_standard_description_5": "alertes",
   "logs_order_selection_details_standard_description_6": "suivi des logs en temps réel.",
-  "logs_order_selection_details_standard_price": "Les 100 premiers Go : {{price}} HT/Go/mois",
+  "logs_order_selection_details_standard_description_7": "rétention au choix 14 jours/1 mois/3 mois/1 an",
```

Maybe we can improve readability:

```suggestion
  "logs_order_selection_details_standard_description_7": "rétention au choix 14 jours / 1 mois / 3 mois / 1 an",
```

Is that make sense for you?